lte/gateway/deploy/agw_install.sh

```diff
@@ -8,7 +8,14 @@ SUCCESS_MESSAGE="ok"
 NEED_REBOOT=0
 WHOAMI=$(whoami)
 KVERS=$(uname -r)
-MAGMA_VERSION="v1.3"
+MAGMA_VERSION="${MAGMA_VERSION:-v1.3}"
```

Comment:
let's not be fancy and just set the MAGMA_VERSION to the current branch. 
The only improvement I see would be to use tags and keep "stable" as the MAGMA_TAG. That would avoid us to go change it every time we bump a new version.

Reply:
I already needed to use this for testing a specific commit when debugging bare metal agw provisioning